addon/components/docs-header/search-result/index.hbs

```diff
@@ -1,12 +1,12 @@
-{{! template-lint-disable no-unknown-arguments-for-builtin-components }}
 <div {{on 'click' this.onClick}} {{on 'mouseenter' this.onMouseEnter}} data-test-search-result>
   <LinkTo
     class={{
       concat
       'docs-block docs-py-2 docs-px-3 docs-text-black docs-no-underline hover:docs-bg-grey-lighter '
       (if @selected 'docs-bg-grey-lighter')
     }}
-    @params={{this.linkArgs}}
+    @route={{this.linkArgs.route}}
```

Comment:
I think I had split these args out like this before and it broke things. Are you sure this works?

Reply:
@rwwagner90 yes, this works in combination with `@models` argument